src/dmx/struct.h

```diff
@@ -102,6 +102,7 @@ typedef struct dmx_driver_t {
     void *context;               // The contexted for the user-side callback.
   } rdm_cbs[RDM_RESPONDER_PIDS_MAX];  // A table containing information on RDM callbacks.
 
+  char device_label[33];
```

Comment:
Does this get used anywhere? I can't find its assignment.

EDIT: it looks like in rdm_register_device_label() the function rdm_pd_alloc() is invoked which would mean that the array declaration isn't needed here.

Reply:
Indeed, this is a leftover from a first iteration. I'll remove it